waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -1183,3 +1201,15 @@ proc mountRlnRelay*(node: WakuNode, conf: WakuNodeConf|Chat2Conf, spamHandler: O
       if res.isErr:
         return err("dynamic rln-relay could not be mounted: " & res.error())
       return ok(true)
+
+
+proc mountRlnRelay*(node: WakuNode, conf: WakuNodeConf|Chat2Conf, spamHandler: Option[SpamHandler] = none(SpamHandler), registrationHandler: Option[RegistrationHandler] = none(RegistrationHandler)): RlnRelayResult[bool] {.raises: [Defect, ValueError, IOError, CatchableError, Exception].} =
```

Comment:
Unrelated to this PR, but procs should not really raise `CatchableError` or `Exception`. If they are raised from an underlying module/procedure they should be caught and translated locally. This can be addressed in future as it's not related to adding metrics. :) See: https://status-im.github.io/nim-style-guide/errors.exceptions.html for more.